src/components/percentage/__snapshots__/percentage.test.jsx.snap

```diff
@@ -0,0 +1,51 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Percentage has defaults 1`] = `
+<body>
+  <div>
+    <div
+      class="container"
+      role="presentation"
+    >
+      <label
+        class="label"
+        for="foo"
+      >
+        the label
+      </label>
+      <div
+        class="control-container"
+      >
+        <input
+          aria-describedby="fooHint"
+          class="input"
+          id="foo"
+          max="100"
+          min="0"
+          step="0.01"
+          type="number"
+          value=""
+        />
+        <div
+          class="icons-container"
+        >
+          <span
+            class="percent-sign"
+          >
+            %
+          </span>
+        </div>
+      </div>
+    </div>
+  </div>
+</body>
+`;
+
+exports[`Percentage has defaults 2`] = `
+Object {
+  "dirty": false,
+  "id": "foo",
+  "name": undefined,
+  "value": NaN,
```

Comment:
I know this is just a snapshot, but is it weird that the default value for a percentage is `NaN` and not `undefined` or something like that?

I mean, it makes sense - we call `parseFloat` on undefined, just noticed it here

Reply:
This might be indicative of something wrong too.

If the user types something, saves it, and then deletes the value, is it NaN? If so, we would try to persist the `value` on the ref and the backend might not be happy.

Reply:
I think we'll want to follow the same API as native inputs. If there is no value, it returns undefined or an empty string. I'm not sure which is best for this scenario -- but less code the better?

Reply:
I decided to standardize on `undefined`, because I think we definitely want to avoid mixing string values as a possibility.